README.md

```diff
@@ -5,43 +5,68 @@
 [![GitHub forks](https://img.shields.io/github/forks/noraj/OSCP-Exam-Report-Template-Markdown?style=flat-square)](https://github.com/noraj/OSCP-Exam-Report-Template-Markdown/network)
 [![GitHub license](https://img.shields.io/github/license/noraj/OSCP-Exam-Report-Template-Markdown?style=flat-square)](https://github.com/noraj/OSCP-Exam-Report-Template-Markdown/blob/master/LICENSE)
 
-I created an **OSCP Exam Report Template in Markdown** so LaTeX, Microsoft Office Word, LibreOffice Writter are no longer needed during your OSCP exam!
-
-Now you can be efficient and faster during your exam report redaction! 
-
-**whoisflynn improved template v3.2**
+## Fork & update/overhaaul of noraj's original script to provide more flexibility & ease in report generation.
```

Comment:
I'm not adding any Joplin specific files to this repository.

Reply:
I don't think there were any truly joplin specific files (just Joplin md exports), but that's fair. Thanks for the review.

Reply:
The JEX files requirements.